source/SAMRAI/tbox/NVTXUtilities.h

```diff
@@ -0,0 +1,71 @@
+/*************************************************************************
+ *
+ * This file is part of the SAMRAI distribution.  For full copyright
+ * information, see COPYRIGHT and LICENSE.
+ *
+ * Copyright:     (c) 1997-2018 Lawrence Livermore National Security, LLC
+ * Description:   Class to record statistics during program execution.
+ *
+ ************************************************************************/
+
+#ifndef included_tbox_NVTXUtilities
+#define included_tbox_NVTXUtilities
+
+#if defined(ENABLE_NVTX_REGIONS) && defined(HAVE_CUDA)
+
+#include "cuda_runtime.h"
+#include "nvToolsExt.h"
+#include "cuda_profiler_api.h"
+
+const uint32_t colors[] = { 0x0000ff00, 0x000000ff, 0x00ffff00, 0x00ff00ff, 0x0000ffff, 0x00ff0000, 0x00ffffff };
+const int num_colors = sizeof(colors)/sizeof(uint32_t);
+
```

Comment:
We need namespace protection in this file so that colors and num_colors aren't fully global symbols.